_dev/.storybook/preview.js

```diff
@@ -72,14 +73,32 @@ window.psFacebookGenerateOpenPopup = (component) => () => {
   }, 3000);
 };
 
+const locales = ['de', 'en', 'es', 'fr', 'it', 'nl', 'pl', 'pt'];
```

Comment:
Should we go one step further and get the languages from _translations.json_ ?
(as they should be all first level entries)

Reply:
yes you're right, that's not difficult to get the keys